tests/integration/test_tiler.py

```diff
@@ -244,7 +309,7 @@ def it_locates_tiles_on_the_slide(
             # no binary mask object passed to locate_tiles
             # default value = BiggestTissueBoxMask
             tiles_location_img = scored_tiles_extractor.locate_tiles(
-                slide, scale_factor=10
+                slide, scale_factor=8
```

Comment:
why change to 8?

Reply:
no real reason, only the ihc slide is really small